plugins/reccobeats.py

```
@@ -0,0 +1,80 @@
+
+RECCO_URL = "https://api.reccobeats.com/v1/track/recommendation"
+DEFAULT_SIZE = 5
+
+
+def _get_spotify_track_id(query, reply):
+    """Use the spotify plugin's search to resolve a song name to a Spotify track ID."""
+    from plugins.spotify import _search
+
+    try:
+        data = _search(query, "track", reply)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)